README.md

```diff
@@ -12,6 +12,7 @@ Reference Implementation (IRI).
 * Check JDK installation and set JAVA_HOME if you wish to specify.
 * Only one GPU can be facilitated with dcurl at the moment.
 * If your platform doesn't support Intel SSE, dcurl would be compiled with naive implementation.
+* For the IOTA hardware accelerator, we integrate [Lampa Lab's Cyclone V FPGA PoW](https://github.com/LampaLab/iota_fpga) into dcurl. Lampa Lab supports soc_system.rbf only for DE10-nano board. You need to synthesize to get soc_system.rbf for using Arrow SoCKit board.
```

Comment:
The reason why you adopted the FPGA implementation of Lampa Lab should be addressed as well.

Reply:
Do you think it is necessary to maintain our own fork for FPGA-based implementation?

Reply:
Yes, it is necessary to do it. As we know, RocketBoards.org provides Golden System Reference Design (GSRD) [0] includes Linux drivers, OS, boot loader and GHRD for Cyclone V SoC. In the future, we need to integrate the OPTEE-related solution and the Mender-related solution into own modified GSRD and rebuild the SD card image. For GHRD, we maybe provide new HDL-implemented PoW for new PoW algorithm and rebuild the RBF. 

[0] [Arria V & Cyclone V Golden System Reference Design(GSRD)](https://rocketboards.org/foswiki/Documentation/AVCVGSRD171)

Reply:
@ajblane, got it. Let's fork the repository from Lampa Lab.